.github/DISCUSSION_TEMPLATE/rfcs.yml

```diff
@@ -15,21 +12,18 @@ body:
   - type: dropdown
     id: area
     attributes:
-      label: Which Powertools for AWS Lambda (Java) utility does this relate to?
+      label: Which area does this RFC relate to?
       options:
         - Tracer
         - Logger
         - Metrics
-        - Middleware factory
         - Parameters
-        - Batch processing
-        - Typing
+        - SQS Large Message Handling
+        - SQS Batch Processing
```

Comment:
Should this be named **SQS** Batch Processing and Large Message handling? The modules can also handle other sources of events.

Reply:
I copy pasted it from other files in this folder, feel free to suggest an edit